<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDSSDlg</class>
 <widget class="QDialog" name="CDSSDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSS Image download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_5">
            <property name="suffix">
             <string>Â°</string>
            </property>
            <property name="minimum">
             <number>-89</number>
            </property>
            <property name="maximum">
             <number>89</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spinBox_7">
            <property name="suffix">
             <string>'</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QSpinBox" name="spinBox_6">
            <property name="suffix">
             <string>&quot;</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Declination</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>R.A.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="suffix">
             <string>h</string>
            </property>
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_3">
            <property name="suffix">
             <string>m</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_4">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="CSignSpinBox" name="spinBox_DecSign"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Download from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size (arc min.)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Orientation of preview frame is only approximate.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Automatic file naming</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Download</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSignSpinBox</class>
   <extends>QSpinBox</extends>
   <header>csignspinbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_DecSign</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>CDSSDlg</receiver>
   <slot>slotChangeVal(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChangeVal(QString)</slot>
 </slots>
</ui>
